snykTask/src/index.ts

```diff
@@ -219,7 +221,7 @@ async function runSnykTest(
       .argIf(taskArgs.organization, `--org=${taskArgs.organization}`)
       .argIf(taskArgs.projectName, `--project-name=${projectNameArg}`)
       .line(taskArgs.additionalArguments)
-      .pipeExecOutputToTool(echoToolRunner, jsonReportOutputPath);
+      .pipeExecOutputToTool(snykToHtmlRunner, jsonReportOutputPath);
```

Comment:
issue: `snyk-to-html` outputs `.html` formatted output. `jsonReportOutputPath` is a `.json` formatted file. This looks like a breaking change?

Reply:
suggestion: the purpose of this bit of code is to do something similar to what the `--json-file-output` flag does in the CLI see docs [here](https://docs.snyk.io/snyk-cli/cli-commands-and-options-summary#options-for-multiple-commands).

Is it possible to use this flag instead?

Reply:
The `--json-file-output` is used as an initial execution. Because `snyk code test` does not generate a json file if there are no issues found, a re-run using `--json` is done to send its no-issues-json from console stdout to be saved to a file location for `snyk-to-html` conversion. Essentially, this code snippet relies on `.pipeExecOutputToTool` 2nd parameter to follow a similar json-to-html conversion workflow. The 1st parameter `snyk-to-html` tool prints html to console stdout which is not used.

Reply:
Question/Suggestion: 
I understand the intension here to write the output of `--json` to a file. The proposed solution does this by using the second parameter of `pipeExecOutputToTool()` the first one is just another tool that needs to be specified but is actually completely ignored. The usage of SnykToHTML here is very confusing, since it is not directly obvious if this call has any side effects or not.
Coming back to the original goal, to safe the out of --json in a file, I would suggest to pick a more straightforward way.

* use [execSync()](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#taskexecsync-) which [returns the stdout](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#toolrunneriexecsyncresult-) of the tool.
* Use [writeFile()](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#taskwritefile-) to store the stdout in the file.

This should solve the issue with echo and have a cleaner more straight forward code.
I hope this makes sense.